src/components/Accordion/_Accordion.scss

```diff
@@ -29,7 +29,7 @@
     }
 
     // Eliminates the double border on accordion sets in ul-lis
-    li & {
+    li:not(:first-of-type) & {
         margin-top: -1px;
```

Comment:
why is there even a negative margin?  haha

Reply:
There's a negative margin on the other items in the list (that are not first) to prevent the double-border in accordion sets.